### ISSUE TITLE
AEAD key compromise

### ISSUE THREAD

**Issue by ekr** [mlswg/mls-architecture#213]
> When an AEAD key is compromised, the adversary has access to a set of AEAD keys
for the same chain and the same epoch, hence can decrypt messages sent using
keys of this chain. An adversary cannot send a message to a group which appears
to be from any valid client since they cannot forge the signature.

This doesn't sound quite right. As I understand the mechanism in S 9.1, if you have the key for a given generation that doesn't allow you to compute other keys unless you also have the ratchet_secret.

**Comment by Bren2010** [comment]
Yes, I believe this should say "When a rachet_key is compromised"

**Comment by beurdouche** [comment]
(Feel free to reopen if that's not good enough)
